src/components/CreateAccount.js

```diff
@@ -16,30 +19,42 @@ class CreateAccount extends Component {
             username: '',
             email: '',
             password: '',
+            usernameIsValid: undefined,
```

Comment:
`isUsernameValid` is probably a bit better name since it starts with `is` which we automatically know is a boolean